### DISCUSSION TITLE
Translation Issue: "report" in German App

### DISCUSSION THREAD

**Discussion by raffisweb** [community/community#84060] [Mobile]
The german translation for "report" (to report sth. criminal) in the burgermenu is "melden". It is accidentially translated as "berichten" (to report in a newspaper). This is misleading.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.143.0 (118103692)
Device: iPhone 13 Pro Max (iOS 17.1.2)
</details>

**Comment by esahmed21** [comment]
Hi @raffisweb 👋 

Thank you for bringing this translation bug to our attention! We'll open an internal ticket to address this bug. Feel free to contact us for any other issues. 

Thanks again for your assistance
